src/ovirtapi.js

```diff
@@ -122,9 +126,9 @@ OvirtApi = {
    * @param disk - disk corresponding to the attachment
    * @returns {} - Internal representation of a single VM disk
    */
-  diskToInternal ({ disk, attachment }) {
-    function toBool (val) {
-      return val && val.toLowerCase() === 'true'
+  diskToInternal ({ disk, attachment }: { disk: Object, attachment: Object }): Object {
+    function toBool (val?: string): boolean {
+      return val !== undefined && val.toLowerCase() === 'true'
```

Comment:
It's because of Flow allow without comparison only boolean type, val is not boolean, so it throw error, so I must to compare it with some value